README.md

```diff
@@ -1 +1,35 @@
-# Vet-Clinic
\ No newline at end of file
+# Vet-Clinic
+
+## Getting Started
+
+This repository includes files with plain SQL that can be used to recreate a database:
+
+- Use [schema.sql](./schema.sql) to create all tables.
+- Use [data.sql](./data.sql) to populate tables with sample data.
+- Check [queries.sql](./queries.sql) for examples of queries that can be run on a newly created database. **Important note: this file might include queries that make changes in the database (e.g., remove records). Use them responsibly!**
+
+## Authors
+
+👤 **Author1**
```

Comment:
- Please, kindly make the authors author and author1 to author as you've worked on the project alone.